.github/styles/Google/Headings.yml

```diff
@@ -24,3 +24,11 @@ exceptions:
   - Visual
   - VS
   - Windows
+  - PHPUnit.xml
```

Comment:
We don't usually add things to the Google developer styleguide files, what's the reasoning here? Usually you override things by adding them to the Vocab/Mautic/accept.txt file or the mautic/features.yml.